portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,35 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//function for servlets
+async function showSomething(){
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.text();
+
+    const servletContainer = document.getElementById('servlet1-container');
+    servletContainer.innerText = textFromResponse;
+}
+
+async function getRandomThoughts(){
+    //fetch function that will get info from /hello
+    const serverResponse = await fetch('/hello');
+
+    //parsing to json
+    const ranInfo = await serverResponse.json();
+    //gets a random element w/Math.random(): Error w/ranInfo.length in web console
+    const ranElement = ranInfo[Math.floor(Math.random()*ranInfo.length)];  
+
+    //connecting with index.html to print array of random info
+    const servletContainer = document.getElementById('servlet2-container');
+    servletContainer.innerText = ranElement;
+    //testing what will print to the web console w/ ranInfo.length: prints string instead of int
+    console.log(ranInfo[Math.floor(Math.random()*ranInfo.length)]);
```

Comment:
If you're done debugging and don't anticipate needing to check these values in the console again, you may want to remove these lines to avoid cluttering up your console output. (It's a very common habit to leave print statements like this in - if you open the console on random websites you'll often see various log messages that were left in by accident.)